python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -75,7 +76,8 @@ def __init__(
         self._asset_graph = asset_graph
         self._logger = logger or logging.getLogger("dagster")
 
-        self._asset_record_cache: Dict[AssetKey, Optional[AssetRecord]] = {}
+        self._batch_asset_record_loader = BatchAssetRecordLoader(self._instance, set())
```

Comment:
because we need to pass a BatchAssetRecordLoader into asset status stuff, we can just use it here in CachingInstanceQueryer too, since it does the same thing